pulser/simulation.py

```diff
@@ -74,6 +75,16 @@ def __init__(self, sequence, sampling_rate=1.0):
         self._build_basis_and_op_matrices()
         self._construct_hamiltonian()
 
+    def draw(self):
+        """Draws the simulation, showing both the input sequence and
+        the effective, constant-by-part (sampled) sequence used in QuTip.
+        """
+
+        # The effective time steps
+        time_slices = (1000*self._times).astype(int)
+
+        draw_sequence(self._seq, time_slices)
```

Comment:
Does this fit?
```suggestion
        draw_sequence(self._seq, (1000*self._times).astype(int))
```
You would avoid defining `time_slices`, one less variable I guess.

Reply:
yes, I would just remove the comment on top then